pages/portal/chapters/create.tsx

```diff
@@ -0,0 +1,152 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import { addChapter } from "requests/Chapter";
+import { Chapter } from 'utils/types';
+
+import Navigation from "components/Portal/Navigation";
+
+const handleSubmit = async (e) => {
+    e.preventDefault();
+    const formData = new FormData(e.target);
+    var chapter: Chapter = await addChapter(formData);
+    //After submitting the form send the user to the chapter edit page
+    var chapterName = String(formData.get("name"));
+    window.location.href = chapterName?.replace(/ /g, "_")
+};
+
+const Chapters: NextPage = () => {
+    return (
+        <div className="container">
+            <Head>
+                <title>MindVersity | Admin Portal</title>
+                <link rel="icon" href="/favicon.ico" />
+            </Head>
```

Comment:
There's a way we can get these on every page by using the _document file. You don't have to change it now, but you can leave them off future pages and we'll take care of adding the _document file and changing all the pages in our refactor.

Reply:
Ok I will for all future pages